cirq-google/cirq_google/study/var_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+import sympy
+
+import cirq_google.study.var as var
+
+
+def test_parameters():
+    param = var.Parameter(path=('test', 'subdir'))
+    s = var.Var(param, range(10), label='repetitions')
+    assert str(s.symbol) == 'repetitions'
+    assert s.descriptor == param
+    assert s.unit is None
+    assert s.keys == ['repetitions']
+
+    param = var.Parameter(path=('test', 'subdir'), idx=4)
```

Comment:
This test only varies from the one above by the `idx` parameter, but it doesn't look like anything about the index is asserted. Is there something specific to this context that we can verify? If not can we dedupe the tests?

Related: it would be more clear and follow Google's unit testing best practices to separate each test into its own method with a name that clearly "summarize[s] the behavior being tested and its expected outcome".